hs_core/hydroshare/hs_bagit.py

```diff
@@ -7,6 +7,10 @@
 from uuid import uuid4
 from mezzanine.conf import settings
 
+from django.core.exceptions import ValidationError
```

Comment:
@ismagilani I don't see ValidationError imported on line 10 is used in this module. Please remove it if it is indeed not used.